packages/facebook-oauth/facebook_server.js

```diff
@@ -1,16 +1,16 @@
 Facebook = {};
 import crypto from 'crypto';
-import { Accounts } from 'meteor/accounts-base';
+import {Accounts} from 'meteor/accounts-base';
```

Comment:
This seems to be some linting happening here. I don't think this is the correct one.